homeassistant/components/tradfri/fan.py

```diff
@@ -67,6 +67,7 @@ def __init__(
         """Initialize a switch."""
         super().__init__(device, api, gateway_id)
         self._attr_unique_id = f"{gateway_id}-{device.id}"
+        self._refresh(device, write_ha=False)
```

Comment:
It's beyond me how the device works without these two added lines in `fan.py`, but it does. Tests, however, do not without them.

What I know is that: (1) we have these lines in the other platform files and (2), it's me who didn't put them here in the first place.

Reply:
This shouldnt be needed:
https://github.com/home-assistant/core/blob/dee843bf6e5ca84a94f336a239f6a6138c4c28e6/homeassistant/components/tradfri/base_class.py#L63-L64

Reply:
Now I see. It's these lines we're after:
https://github.com/home-assistant/core/blob/dee843bf6e5ca84a94f336a239f6a6138c4c28e6/homeassistant/components/tradfri/fan.py#L174-L175

Reply:
Ok.

Reply:
Tests still run if I remove the line you suggested. Removing `self._refresh(device, write_ha=False)` when initialising the class, however, breaks tests.